espnet/nets/beam_search.py

```diff
@@ -352,6 +354,8 @@ def forward(
         # set length bounds
         if maxlenratio == 0:
             maxlen = x.shape[0]
+        elif maxlenratio < 0:
```

Comment:
Could you also consider adding a unit test for this modification?

Reply:
Yes, sure. Is it fine now, or it should be done another way?

Reply:
LGTM! Thank you.